conjur/argument_parser/_list_parser.py

```diff
@@ -103,8 +104,8 @@ def _add_list_options(list_subparser: ArgparseWrapper):
                                        '(e.g.  [group:]conjur-root-admins),'
                                        'or passed in the --kind option')
         list_options.add_argument('-p', '--privilege',
-                                  action='store', metavar='VALUE', dest='permitted_members_of',
-                                  help='Mandatory - when combined with --permitted-members-of'
+                                  action='store', metavar='VALUE', dest='privilege',
+                                  help='Mandatory - when combined with --permitted-roles'
                                        'Specifies the roles permitted '
                                        'to exercise this privilege are shown '
                                        '(read | execute | update).')
```

Comment:
@shulifink @szamir1 Please review text

Reply:
Let's discuss....

Reply:
"Lists the roles that have the specified privilege"